imports/plugins/custom/product-variant/client/components/customProductGridItem.js

```diff
@@ -2,6 +2,7 @@ import React from "react";
 import { FacebookShareButton, FacebookIcon, TwitterShareButton, TwitterIcon, WhatsappShareButton, WhatsappIcon } from "react-share";
 import ReactStars from "react-stars";
 import { formatPriceString } from "/client/api";
+import { Components } from "@reactioncommerce/reaction-components";
```

Comment:
line 5 and 6 should be combined because they are imported from same package.

Reply:
Thank you for spotting this. I'll fix it. Any other thing?